operators/README.md

```diff
@@ -58,6 +58,7 @@ Based on [Kubebuilder 2.3](https://github.com/kubernetes-sigs/kubebuilder.git),
 Upon the creation of a *Instance*, the operator triggers the creation of the following components:
 * Kubevirt VirtualMachine Instance and the logic to access the noVNC instance inside the VM (Service, Ingress)
 * An instance of [Oauth2 Proxy](https://github.com/oauth2-proxy/oauth2-proxy) (Deployment, Service, Ingress) to regulate access to the VM.
+* Persistent VirtualMachine Instance based on DataVolume and the logic to access to it
```

Comment:
```suggestion
* A persistent VirtualMachine Instance based on DataVolume and the logic to access to it
```
Is a "persistent VirtualMachine Instance" a well-known concept in CrownLabs? Or did you define it? In the latter, it should be introduced somewhere.
What is a "DataVolume"? Be more verbose, or a future student would be lost.

Reply:
The persistent vm is a normal crownlabs vm, but is based on a datavolume that is a custom resource of the cdi which allow the creation of the pvc, that will receive the image of the vm through an importer pod